### PR TITLE
[BE-1122] Fix TypeError [No Mergey] 

### PR DESCRIPTION
**Pull request by ARtoriouSs**
[Related PR](https://github.com/roadie-app/roadie-server/pull/1961)

Do not merge till it will be tested on this branch

### CODE REVIEW

File: lib/google_directions_api/directions.rb [hunk 1]
```diff
@@ -55,43 +55,31 @@ def data
...
+
+      seconds
     end
 
     def tolls_along_route?
-      if waypoints_present?
-        data["routes"][0]["legs"].each do |leg|
-          next unless leg["steps"].any?
-
-          if leg["steps"]["html_instructions"].try(:downcase).try(:include?, 'toll road')
```
Comment (d-mart) [inline lib/google_directions_api/directions.rb]: Note to self: The original problem was here - `html_instructions` sub-element is an array (see line 93 for correct usage)

File: lib/google_directions_api/directions.rb [hunk 2]
```diff
@@ -55,43 +55,31 @@ def data
...
-        end
-        return meters
-      else
-        data["routes"][0]["legs"][0]["distance"]["value"]
+      meters = 0
+      data["routes"][0]["legs"].each do |leg|
+        meters += leg["distance"]["value"]
       end
+
+      meters
```
Comment (mcsoup) [inline lib/google_directions_api/directions.rb]: Fill like there is some creativity that could be done with the inject method to cut down on the number of lines

File: lib/google_directions_api/directions.rb [hunk 3]
```diff
@@ -55,43 +55,31 @@ def data
...
-        seconds = 0
-        data["routes"][0]["legs"].each do |leg|
-          seconds += leg["duration"]["value"]
-        end
-        return seconds
-      else
-        data["routes"][0]["legs"][0]["duration"]["value"]
+      seconds = 0
+      data["routes"][0]["legs"].each do |leg|
+        seconds += leg["duration"]["value"]
```
Comment (mcsoup) [inline lib/google_directions_api/directions.rb]: Same as above

File: lib/google_directions_api/directions.rb (hunk 2 above)
Comment (ARtoriouSs) [reply lib/google_directions_api/directions.rb]: Agree, done

**Pull request merged**